.github/workflows/fuzzing-ci.yaml

```diff
@@ -50,7 +50,7 @@ jobs:
         sudo apt-get -y install software-properties-common
         sudo add-apt-repository -y "deb http://openresty.org/package/ubuntu $(lsb_release -sc) main"
         sudo apt-get update
-        sudo apt-get install -y git openresty curl openresty-openssl111-dev unzip make gcc libldap2-dev
+        sudo apt-get install -y git openresty curl openresty-openssl111-dev unzip make gcc libldap2-dev libxml2-dev libxslt-dev libssl-dev
```

Comment:
I don't like this change, it greatly increases the difficulty of building a user's local development environment.